src/test/ui/feature-gates/feature-gate-cfg-attr-multi-bootstrap-2.rs

```diff
@@ -0,0 +1,9 @@
+// Test that settingt the featute gate while using its functionality doesn't error.
+// Specifically, if there's a cfg-attr *before* the feature gate.
+
+// compile-pass
+
+#![cfg_attr(all(),)]
+#![cfg_attr(all(), feature(cfg_attr_multi), crate_type="bin")]
```

Comment:
This is a fun example.
I didn't know that crate attributes are configured twice to get the correct feature list early.